File: Services/GatewayApi/FliGen.Services.Api/Models/Leagues/League.cs
﻿using System.Collections.Generic;

namespace FliGen.Services.Api.Models.Leagues
{
    public class League
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public LeagueType LeagueType { get; set; }
        public IEnumerable<PlayerWithLeagueStatus> PlayersLeagueStatuses { get; set; }
    }
}

File: Services/Web/FliGen.Web/ClientApp/src/app/common/league.ts
import { LeagueType } from "./leagueType";
import { PlayerWithLeagueStatus } from "./playerWithLeagueStatus";

export class League{
  id: number;
  name: string;
  description: string;
  leagueType: LeagueType;
  playersLeagueStatuses: PlayerWithLeagueStatus[];
}
